src/components/user-defined-sets/SetsManager.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import CurrentSetManager from './CurrentSetManager';
```

Comment:
The other `Manager` we have doesn't really have a rendering, except for the please wait while layers are loading. For you, is there a similarity?

Reply:
I kind of just meant Manager as in the component would keep track of the sets of cells via creating, editing their names, and deleting, but would leave other "set activities" to a component like `SetsOperator` to do unions, intersections, etc.

Reply:
Maybe rename `LayerManagerPublisher`? just `LayerPublisher`? `DataLayerPublisher`? "Manager" is very generic, and that's fine, but I want to avoid suggesting a relationship.